modules/primer-dropdown/README.md

```diff
@@ -0,0 +1,333 @@
+# Primer CSS dropdown
+
+[![npm version](http://img.shields.io/npm/v/primer-dropdown.svg)](https://www.npmjs.org/package/primer-dropdown)
+[![Build Status](https://travis-ci.org/primer/primer-css.svg?branch=master)](https://travis-ci.org/primer/primer-css)
+
+This repository is a submodule of the [Primer monorepo][primer].
+
+## Install
+
+This repository is distributed with [npm]. After [installing npm][install-npm], you can install `primer-dropdown` with this command.
+
+```
+$ npm install --save primer-dropdown
+```
+
+## Usage
+
+The source files included are written in [Sass][] with SCSS syntax. If you add your `node_modules` directory to your Sass `include-path`, you can import the source files with:
+
+```scss
+@import "primer-dropdown/index.scss";
+```
+
+You can also import specific portions of the module by importing those partials from the `/lib/` folder. **You also need to install [primer-support] and import it** with:
+
+```scss
+@import "primer-support/index.scss";
+```
+
+## Build
+
+The npm `build` script outputs a CSS version to `build/build.css`, which is distributed with the npm package. You can build it manually with:
+
+```
+$ npm run build
+```
+
+## Documentation
+
+<!-- %docs
+title: Dropdown
+status: Stable
+-->
+
+Dropdowns are lightweight, JavaScript-powered context menus for housing navigation and actions. They're great for instances where you don't need the full power (and code) of the select menu.
+
+{:toc}
+
+## Basic examples
+
+Dropdowns may be used with `<details>` and `<summary>` elements or JS behaviors and a `<button>`. **[Each dropdown menu requires a directional class](#alignment)**, much like our tooltips.
+
+Using `<details>` and `<summary>` (no JS required!):
+
+```html title="Dropdown from details"
+<details class="details-reset dropdown">
+  <summary class="btn dropdown-toggle" type="button" aria-expanded="false" aria-haspopup="true">
+    Dropdown
+    <div class="dropdown-caret"></div>
+  </summary>
+
+  <div class="dropdown-menu-content">
```

Comment:
`.dropdown-menu-content` would hide the content by default. We have been using it without `.dropdown-menu-content` when using it with `<details>`.